src/encoded/static/components/filegallery.js

```diff
@@ -2542,7 +2784,7 @@ class FileGalleryRendererComponent extends React.Component {
                         >
                             <TabPanelPane key="browser">
                                 <GenomeBrowser
-                                    files={includedFiles}
+                                    files={graphIncludedFiles}
```

Comment:
I'm not sure that this change quite works the way it should. Trying to remember why there are so many different arrays of files and what they are all for took a little while though - really, when I first wrote all this code I should have documented this better. It seems like you want to make sure that you're allowing the user to choose an analysis, just like for the graph, which is very important. However, I think that the difference between "includedFiles" and "graphIncludedFiles" lies in whether or not the array of files is filtered based on the user-selected filters (see lines 2444 and 2465) and it seems like it might be important to keep this filtering on the genome browser. After looking at the code, I compared the demo and production and if you select filters on the graph tab and then switch to the genome browser tab, the filters are not applied to the displayed files, and I think that is because of this line change.

![Screen Shot 2020-04-28 at 11 28 06 AM](https://user-images.githubusercontent.com/2887343/80524269-30ef6c00-8944-11ea-8a7d-99461221e13f.png)

Reply:
Switching between tabs was a nightmare in this ticket. I’ll try to figure out what’s happening here.

Reply:
@emro I patched the demo directly to change `graphIncludedFiles` back to `includedFiles`. Does it look better to you? I’m not sure why I changed that.

https://encd-5116-pipeline-graph-pr1-fytanaka.demo.encodedcc.org/experiments/ENCSR000AFA/

Reply:
The update looks good in the code but for some reason I'm getting a "bad gateway" error when I try to visit your demo. The demo says it's running on AWS, though.

Reply:
I changed the domain name this morning so you’ll have to go to https://encd-5116-pipeline-graph-pr2-fytanaka.demo.encodedcc.org/ now.